patronload/patron.py

```diff
@@ -19,83 +19,69 @@
 logger = logging.getLogger(__name__)
 
 
-def create_and_write_to_zip_file_in_memory(
-    xml_file_name: str, file_content: str
-) -> BytesIO:
-    """
-    Create zip file in memory and zip a string value.
-
-    Used to zip patron XML data before uploading to S3 bucket.
-
-    Args:
-        xml_file_name: The name of the XML file to be zipped.
-        file_content: The file content to be zipped, must be a str.
-    """
-    zip_file_object = BytesIO()
-    with ZipFile(zip_file_object, "w") as zip_file:
-        zip_file.writestr(
-            xml_file_name,
-            file_content,
-        )
-        return zip_file_object
-
-
-def format_phone_number(phone_number: str) -> str:
-    """
-    Format a string of 10 numbers as a phone number with dashes.
-
-    Args:
-        phone_number: A string that may contain a phone number.
-    """
-    return re.sub(r"(\d{3})(\d{3})(\d{4})", r"\1-\2-\3", phone_number)
-
-
-def populate_common_fields(
-    patron_template: BeautifulSoup,
-    patron_dict: dict[str, Any],
-    six_months: str,
-    two_years: str,
-) -> BeautifulSoup:
-    """
-    Populate the fields common to both staff and student patron records.
-
-    The order of the patron record fields must be in same order as they in the
-    STAFF_FIELDS and STUDENT_FIELDS value lists.
+def patrons_xml_string_from_records(
```

Comment:
Not a comment on the maintenance work -- this refactor and reordering makes sense! -- but it did take me a bit to understand from the diff (had to compare two branches by toggling between them) that this was just ordering, and not a meaningful change to code.

Reply:
Sorry that's why I mentioned it in the commit message but in retrospect, it should have been a separate commit

Reply:
Apologies are mine!  I missed that in the commit message; think that's more than sufficient.